### ISSUE TITLE
SDN - Investigate vSphereOpenshiftClusterHealthFail errors from AlertManager in KLAB2

### ISSUE THREAD

**Issue by wmhutchison** [BCDevOps/developer-experience#4289]
**Describe the issue**
Due to expired AlertManager alerts going away far too fast, it's not certain if this is a known issue whose silence expired during Openshift upgrades to 4.12, or if this is a net-new issue. Not a high priority since this alert is for integration required if we were to make use of ESXi storageclass (we do not as we use NetApp). Still, would be good to dig into this as time permits and resolve.

**Additional context**
Add any other context, attachments or screenshots

**How does this benefit the users of our platform?**

**Definition of done**
- [x] Investigate with VMware team as needed to determine if this is an upgrade issue, or something that can be ignored.
- [x] If the former, then take steps to resolves.
- [ ] If the latter, then put into a long-term Silence that's then documented.

**Comment by wmhutchison** [comment]
Link to OCP 4.12 requirements for vSphere account permissions. https://docs.openshift.com/container-platform/4.12/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-installer-infra-requirements_installing-vsphere

**Comment by wmhutchison** [comment]
Updated link for OCP 4.14: https://docs.openshift.com/container-platform/4.14/installing/installing_vsphere/upi/upi-vsphere-installation-reqs.html#installation-vsphere-installer-infra-requirements_upi-vsphere-installation-reqs

**Comment by StevenBarre** [comment]
4.14 requires additional permissions for the service account that the vsphere operator uses to talk to vcenter. Added those extra things and its has stopped complaining.

Also ran the fix in https://access.redhat.com/solutions/7057384
```
oc patch clustercsidriver --subresource=status csi.vsphere.vmware.com  --type='merge' -p '{"status": {"conditions": [ {"type": "VMwareVSphereControllerDummyProgressing", "status": "False", "reason": "Forced"}]}}'
```

**Comment by StevenBarre** [comment]
https://github.dxc.com/AdvSol/mcs-ansible-tower/pull/27 adds the vCenter CA certs to the util servers so we can run `openshift-install` interactively and generate the `install-config.yaml` with updated values.

**Comment by StevenBarre** [comment]
Waiting on RITM0192009

**Comment by StevenBarre** [comment]
Firewall change scheduled for Monday night

**Comment by StevenBarre** [comment]
https://github.com/bcgov-c/rhcos-ignition-builder/pull/146

vSphere operator is happy now in KLAB2. Will fix Emerald during next patching cycle as a node reboot will be required.
